docs/source/dask.rst

```diff
@@ -0,0 +1,98 @@
+Using with Dask
+===============
+
+``UCX/UCX-PY`` can be used with `Dask <https://dask.org/>`_ as a drop-in replacement for the communication protocol between workers.  Below we show how to use UCX-PY with both helper utilities such as `dask-cuda <https://github.com/rapidsai/dask-cuda>`_
+and manually starting a dask cluster with UCX enabled.  Additionally, we demonstrate using UCX with a `cuDF Example`_ and `CuPy Example`_
+
+Starting with Dask-cuda
+-----------------------
+
+`dask-cuda <https://github.com/rapidsai/dask-cuda>`_ is Python tool designed to help with cluster deployment and management of Dask workers on CUDA-enabled systems.  Dask-cuda can be used to in two ways: inline and CLI.
+
+Inline
+~~~~~~
+
+.. code-block:: python
+
+    from dask.distributed import Client
+    from dask_cuda import DGX
+    from dask_cuda.initialize import initialize
+
+    # ON/OFF settings for various devices
+    enable_tcp_over_ucx = True
+    enable_infiniband = False
+    enable_nvlink = False
+
+
+    # initialize client with the same settings as workers
+    initialize(
+        create_cuda_context=True,
+        enable_tcp_over_ucx=enable_tcp_over_ucx,
+        enable_infiniband=enable_infiniband,
+        enable_nvlink=enable_nvlink,
+    )
+
+    cluster = DGX(
+        interface="enp1s0f0",
+        protocol="ucx",
+        enable_tcp_over_ucx=enable_tcp_over_ucx,
+        enable_infiniband=enable_infiniband,
+        enable_nvlink=enable_nvlink,
+    )
+    client = Cluster(client)
```

Comment:
Maybe add a small note to explain that the interface must be the ethernet interface.